static/js/jira-configuration.js

```diff
@@ -68,38 +68,52 @@ $(".delete-connection-link").click(function(event) {
 	});
 });
 
-$(".sync-connection-link").click(function(event) {
-	event.preventDefault();
+const initializeBackfillDateInput = function () {
+	const dateElement = document.getElementById('backfill-date-picker');
+	const date = new Date();
+	date.setFullYear(date.getFullYear() - 1);
+	dateElement.value = date.toISOString().split('T')[0];
+	dateElement.max = new Date().toISOString().split('T')[0];
```

Comment:
Should we set a `dateElement.min` for UX purpose?

Reply:
not necessary